doc/users/testsuite.html

```diff
@@ -247,6 +247,16 @@
 100% tests passed, 0 tests failed out of 2
 
 Total Test time (real) =   0.43 sec</pre>
+    In some cases, additional input data has to be provided in users' tests. For
+    example, we may have an input file
+      <pre>
+tests/support/my_test_2.input</pre>
+    which shall be parsed within <code>my_test_2.cc</code>. To read this file,
+    one can use the <code>SOURCE_DIR</code> preprocessor variable which will be equal
+    to the parent folder of each <code>.cc</code> test. In other words, given the current
+    imaginary directory stucture, <code>const std::string input = SOURCE_DIR + "/my_test_2.input"</code>
```

Comment:
Actually, this would not work as you cant add `char[]`s. One can use the following instead.
```
const std::string str = std::string(SOURCE_DIR) + "/my_test_2.input";
```

Reply:
or simply omit the `+`, i.e.
```
const std::string input = SOURCE_DIR "/my_test_2.input";
```

Reply:
good point, fill fix...

Reply:
i opted for the first variant as I personally find it easier to read and extend if needed.